Dockerfile.test.gpu

```diff
@@ -52,28 +54,33 @@ RUN CUDNN_MAJOR=$(cut -d '.' -f 1 <<< "${CUDNN_VERSION}"); \
         build-essential \
         g++-${GPP_VERSION} \
         moreutils \
-        libcudnn${CUDNN_MAJOR}=${CUDNN_VERSION} \
-        libnccl2=${NCCL_VERSION_OVERRIDE} \
-        libnccl-dev=${NCCL_VERSION_OVERRIDE}
+        libcudnn${CUDNN_MAJOR}=${CUDNN_VERSION} && \
+    rm -rf /var/lib/apt/lists/*
 
 # setup ssh service
 RUN ssh-keygen -f /root/.ssh/id_rsa -q -N ''
 RUN cp -v /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys
 
-# Install Python.
-RUN apt-get update -qq && apt-get install -y python${PYTHON_VERSION} python${PYTHON_VERSION}-dev python${PYTHON_VERSION}-distutils
-RUN ln -s -f /usr/bin/python${PYTHON_VERSION} /usr/bin/python
-RUN ln -s -f /usr/bin/python${PYTHON_VERSION} /usr/bin/python${PYTHON_VERSION/%.*/}
-RUN wget --progress=dot:mega https://bootstrap.pypa.io/get-pip.py && python get-pip.py && rm get-pip.py
+RUN wget -q https://repo.continuum.io/miniconda/Miniconda3-py38_4.12.0-Linux-x86_64.sh -O miniconda.sh && \
```

Comment:
Do we really want to base our test images on miniconda? Why is this needed?

Reply:
Unfortunately, this is required for [installing nvtabular](https://pypi.org/project/nvtabular/).  There is an option to install it w/ only CPU support (via pip), so maybe that would be a reasonable option for CI/CD purposes?  But for end-user runtimes, they'd probably want the full GPU support.

Reply:
Now removed conda and nvtabular from the Dockerfile.test.* images in favor of the `docker/horovod-nvtabular/Dockerfile`.